### PR TITLE
Add fi_FI translations.

### PR DESCRIPTION
**Pull request by fabianmakila**
Added Finnish translation.

### CODE REVIEW

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Olet jo yhdistänyt tälle palvelimelle!
+velocity.error.already-connected-proxy=Olet jo yhdistänyt tälle välityspalvelimelle!
```
Comment (Chicken) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: "Yhdistänyt" is more of a past tense.
```suggestion
velocity.error.already-connected=Olet jo yhteydessä tälle palvelimelle!
velocity.error.already-connected-proxy=Olet jo yhteydessä tälle välityspalvelimelle!
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+velocity.error.already-connecting=Yrität jo tällä hetkellä yhdistää palvelimeen!
+velocity.error.cant-connect={0} ei saatu yhteyttä: {1}
+velocity.error.connecting-server-error=Yhteyttä palvelimeen {0} ei voitu muodostaa. Yritä myöhemmin uudelleen.
+velocity.error.connected-server-error=Yhteydessäsi palvelimeen {0} tapahtui virhe.
+velocity.error.internal-server-connection-error=Virhe tapahtui sisäisessä palvelinyhteydessä.
+velocity.error.logging-in-too-fast=Kirjaudut sisään liian nopeasti, yritä uudelleen hetken kuluttua.
+velocity.error.online-mode-only=Et ole kirjautuneena Minecraft tilillesi. Jos olet kirjautuneena sisään, yritä pelin uudelleenkäynnistämistä.
```
Comment (Chicken) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: Not so sure about this one. But shouldn't it be "Minecraft-tilillesi" (notice the -)

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Olet jo yhdistänyt tälle palvelimelle!
+velocity.error.already-connected-proxy=Olet jo yhdistänyt tälle välityspalvelimelle!
+velocity.error.already-connecting=Yrität jo tällä hetkellä yhdistää palvelimeen!
+velocity.error.cant-connect={0} ei saatu yhteyttä: {1}
+velocity.error.connecting-server-error=Yhteyttä palvelimeen {0} ei voitu muodostaa. Yritä myöhemmin uudelleen.
+velocity.error.connected-server-error=Yhteydessäsi palvelimeen {0} tapahtui virhe.
+velocity.error.internal-server-connection-error=Virhe tapahtui sisäisessä palvelinyhteydessä.
```
Comment (Chicken) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: Sounds just wrong.
```suggestion
velocity.error.internal-server-connection-error=Tapahtui palvelimen sisäinen yhteysvirhe.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+velocity.error.logging-in-too-fast=Kirjaudut sisään liian nopeasti, yritä uudelleen hetken kuluttua.
+velocity.error.online-mode-only=Et ole kirjautuneena Minecraft tilillesi. Jos olet kirjautuneena sisään, yritä pelin uudelleenkäynnistämistä.
+velocity.error.player-connection-error=Yhteydessäsi tapahtui sisäinen virhe.
+velocity.error.modern-forwarding-needs-new-client=Tämä palvelin on yhteensopiva vain version 1.13 ja ylemmän kanssa.
```
Comment (Chicken) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: "ja" is the wrong word to use here and "ylemmän" sounds unnatural
```suggestion
velocity.error.modern-forwarding-needs-new-client=Tämä palvelin on yhteensopiva vain version 1.13 tai uudemman kanssa.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+velocity.error.modern-forwarding-failed=Valitsemasi palvelin ei lähettänyt välityspyyntöä välityspalvelimelle. Tarkista että palvelin on määritetty oikein Velocityä varten.
+velocity.error.moved-to-new-server=Sinut potkittiin pois palvelimelta {0}: {1}
+velocity.error.no-available-servers=Yhtään palvelinta ei ole tällä hetkellä saatavilla. Yritä myöhemmin uudelleen tai ota yhteyttä palvelimen ylläpitäjään.
+
+# Commands
+velocity.command.generic-error=Virhe tapahtui komentoa suorittaessa.
```
Comment (Chicken) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: Sounds so unnatural.
```suggestion
velocity.command.generic-error=Komentoa suorittaessa tapahtui virhe.
```

**Review by Chicken — CHANGES_REQUESTED** [review]
Overall looking good though!

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties (hunk 2 above)
Comment (fabianmakila) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties]: You are entirely correct, I will push a commit shortly.

**Review by Chicken — APPROVED** [review]

**Comment by astei** [comment]
Thanks! In the future, we will be performing translations on [Crowdin](https://crowdin.com/project/velocity).

**Pull request merged**